DataFormats/ForwardDetId/interface/HFNoseDetId.h

```diff
@@ -26,6 +26,7 @@ class HFNoseDetId : public DetId {
   static const int HFNoseCoarseN = 8;
   static const int HFNoseFineTrigger = 3;
   static const int HFNoseCoarseTrigger = 2;
+  static const int kHFNoseLayerEEmax = 6;
```

Comment:
the other public consts don't start with `k`, change this one to fit the style?

Reply:
done